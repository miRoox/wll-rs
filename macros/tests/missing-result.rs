use wll_macros::{wll_export, wll_setup, wll_teardown};

#[wll_setup]
fn setup() {}

#[wll_teardown]
fn teardown() {}

#[wll_export]
fn add_two(a: isize, b: isize) -> isize {
    a + b
}

fn main() {}
