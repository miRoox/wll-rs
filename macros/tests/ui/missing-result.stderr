error[E0308]: mismatched types
  --> $DIR/missing-result.rs:10:35
   |
10 | fn add_two(a: isize, b: isize) -> isize {
   | ----------                        ^^^^^ expected `isize`, found enum `std::result::Result`
   | |
   | this expression has type `isize`
   |
   = note: expected type `isize`
              found enum `std::result::Result<_, _>`
